helm/templates/minio-network-policy.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: minio
+spec:
+  podSelector:
+    matchLabels:
+      app: minio
+  policyTypes:
+  - Ingress
+  - Egress
+  ingress:
+  - from:
+    - ipBlock:
+        cidr: 0.0.0.0/0
```

Comment:
I think it should allow access only from minio pod

Reply:
I believe that `minio` is accepting traffic from `nginx` on 8080.
The [lines](https://github.com/CloudWebManage/cwm-worker-deployment-minio/pull/18/files#diff-5aec091dd571aefe0a6bbf67ffee8b1b1baf2a50745749b157196d806e1dcdd7R16-R18) below allow traffic from `nginx`.

Reply:
please read how the from rules are handled, if you want to limit to traffic from a specific pod you don't need to specify ipBlock 0.0.0.0